Purchases/Networking/Operations/Handling/SimpleDataResponseHandler.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SimpleDataResponseHandler.swift
+//
+//  Created by Joshua Liebowitz on 11/30/21.
+
+import Foundation
+
+class SimpleDataResponseHandler {
```

Comment:
Maybe `NoContentResponseHandler`? `Simple` doesn't seem descriptive enough here 🤷‍♂️ 

```suggestion
class NoContentResponseHandler {
```

Reply:
Done! Best new name